src/main/scala/com/ubirch/webui/models/keycloak/util/BareKeycloakUtil.scala

```diff
@@ -634,7 +692,7 @@ case class MemberResourceRepresentation(resource: UserResource, representation:
     * When a new device is created, get which password should be his:
     * 1 If the user belongs to a DEFAULT_PASSWORD group, select the password defined in the group's attributes
     * 2 If not, get the user's DEFAULT_DEVICE_PASSWORD attribute
-    * If it doesn't exist, create it and use it
+    * If it doesn't exist, create a random one
```

Comment:
Ah funny, so it is either default or random password. The default one would be defined by the group or by whome?

Reply:
The default one is defined by the groups. If a user is part of such a group, all their devices will have this default one. Otherwise, it'll be a random one.

Right now, there's no way for a user to join such a group. It has to be done through keycloak.